.github/workflows/release_deb.yml

```diff
@@ -0,0 +1,112 @@
+name: "Release DEB package"
+
+on:
+  release:
+    types:
+      - published
+env:
+  QT_VERSION: 5.15.0
+jobs:
+  aur:
+    name: "Release DEB package"
+    runs-on: "ubuntu-16.04"
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          fetch-depth: 0
+
+      - name: Get version
+        run: |
+          echo ::set-env name=VERSION::$(echo $(git describe --tags --long | cut -d- -f1 | sed 's/^v//'))
+          a=${VERSION%.*}.
+          b=$(<.ci/STABLE_VERSION)
+          if [[ "$a" == "$b" ]]; then
+            echo ::set-env name=ISBETA::false
+          else
+            echo ::set-env name=ISBETA::true
+          fi
+                 
+      - name: Install Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      
+      - name: Restore Qt from cache
+        id: cache-qt
+        uses: actions/cache@v1
+        with:
+          path: ../Qt
+          key: Qt-ubuntu-16.04-${{ env.QT_VERSION }}
+
+      - name: Set up Qt environment
+        uses: jurplel/install-qt-action@v2
+        with:
+          cached: ${{ steps.cache-qt.outputs.cache-hit }}
+          version: ${{ env.QT_VERSION }}
+
+      - name: Build
+        run: |
+          mkdir build
+          cd build
+          cmake .. -DCMAKE_BUILD_TYPE=Release
+          cmake --build . --config Release -j2
+          
+      - name: Deploy
+        run: |
+          sudo apt install libxcb*
+          cd build/
+          wget -c "https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-x86_64.AppImage"
+          chmod a+x linuxdeployqt*.AppImage
+          ./linuxdeployqt*.AppImage ./cpeditor -qmake=../../Qt/${{ env.QT_VERSION }}/gcc_64/bin/qmake
+          
+      - name: Preparing application files' directory
+        run: |
+          mkdir cpeditor_${VERSION}_amd64 && cd cpeditor_${VERSION}_amd64
+          mkdir usr && cd usr
```

Comment:
These mkdirs can be reduced by using -p flag and [path1, path2] bash script notations.